src/components/Toggle.js

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useState } from 'react';
+import { Icon } from '@iconify/react';
+import sunIcon from '@iconify/icons-feather/sun';
+import moonIcon from '@iconify/icons-feather/moon';
+import { setTheme } from '../utils/theme';
+import './Toggle.css';
+
+function Toggle()
+{
+    const [togClass, setTogClass] = useState('dark');
+    let theme = localStorage.getItem('theme');
+
+    const handleOnClick = () =>
+    {
+        if(localStorage.getItem('theme') === 'theme-dark')
+        {
+            setTheme('theme-light');
+            setTogClass('light')
+        }
+        else
+        {
+            setTheme('theme-dark');
+            setTogClass('dark')
+        }
```

Comment:
```suggestion
        storagedTheme = localStorage.getItem('theme')
        setTheme(storagedTheme)
        setTogClass(storagedTheme)
```

Reply:
This change would require if else statements, because we are fetching the theme from local storage and putting the opposite theme that was in local storage, unlike the cases previously where the exact same theme was set to the state variables. The previous cases, I have changed.